cpp/src/parquet/metadata.h

```diff
@@ -173,8 +173,13 @@ class PARQUET_EXPORT FileMetaData {
 
   // file metadata
   uint32_t size() const;
+
   int num_columns() const;
+  std::vector<int> AllColumnIndices() const;
+
   int64_t num_rows() const;
+  std::vector<int> AllRowGroups() const;
```

Comment:
I don't know that these functions make sense as public APIs -- I added them internally as a convenience. Can we instead expose the `Arange` (with a better/more descriptive name) in a `util` namespace somewhere and spell out `Arange(metadata->num_columns())` where we need that?

Reply:
will do